polkadot/runtime/westend/src/lib.rs

```diff
@@ -1874,7 +1874,6 @@ pub mod migrations {
 		parachains_shared::migration::MigrateToV1<Runtime>,
 		parachains_scheduler::migration::MigrateV2ToV3<Runtime>,
 		pallet_staking::migrations::v16::MigrateV15ToV16<Runtime>,
-		pallet_staking::migrations::v17::MigrateV16ToV17<Runtime>,
```

Comment:
Unfortunately we have already upgraded westend to V17.. you can check in the chain state. 

What did this migration do? We need to see if any special consideration is needed for it or not.